modules/data.atmosphere/R/GEFS_helper_functions.R

```diff
@@ -73,37 +63,12 @@ noaa_grid_download <- function(lat_list, lon_list, forecast_time, forecast_date,
   
   model_dir <- file.path(output_directory, model_name_raw)
   
+  #Availability: most recent 4 days
   curr_time <- lubridate::with_tz(Sys.time(), tzone = "UTC")
   curr_date <- lubridate::as_date(curr_time)
-  
-  noaa_page <- readLines('https://nomads.ncep.noaa.gov/pub/data/nccf/com/gens/prod/')
-  
-  potential_dates <- NULL
-  for(i in 1:length(noaa_page)){
-    if(stringr::str_detect(noaa_page[i], ">gefs.")){
-      end <- stringr::str_locate(noaa_page[i], ">gefs.")[2]
-      dates <- stringr::str_sub(noaa_page[i], start = end+1, end = end+8)
-      potential_dates <- c(potential_dates, dates)
-    }
-  }
-  
-  
-  last_cycle_page <- readLines(paste0('https://nomads.ncep.noaa.gov/pub/data/nccf/com/gens/prod/gefs.', dplyr::last(potential_dates)))
-  
-  potential_cycle <- NULL
-  for(i in 1:length(last_cycle_page)){
-    if(stringr::str_detect(last_cycle_page[i], 'href=\"')){
-      end <- stringr::str_locate(last_cycle_page[i], 'href=\"')[2]
-      cycles <- stringr::str_sub(last_cycle_page[i], start = end+1, end = end+2)
-      if(cycles %in% c("00","06", "12", "18")){
-        potential_cycle <- c(potential_cycle, cycles)
-      }
-    }
-  }
```

Comment:
`potential_cycle` was never used below, so this entire URL request and loop had no effect.